src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIRequestSmelter.java

```diff
@@ -272,6 +302,10 @@ private IAIState checkIfAbleToSmelt(final int amountOfFuel, final boolean checkS
             }
         }
 
+        if(furnaceBurning)
+        {
+            //Todo: Scalable Delay 
```

Comment:
for the scalable Delays you can either use setDelay(ticks) which pauses the AI for X ticks, or add some local counter var.

Reply:
Yeah, I actually had some in there, but removed them and put the todos in to mark where I wanted them in the next phase. That's phase is where I work the primary and secondary stats into the behavior. Better parallelization on primary, and more responsive on secondary (thus the scalable delays)